pkg/api/entries.go

```diff
@@ -127,26 +127,22 @@ func GetLogEntryByIndexHandler(params entries.GetLogEntryByIndexParams) middlewa
 	return entries.NewGetLogEntryByIndexOK().WithPayload(logEntry)
 }
 
-// CreateLogEntryHandler creates new entry into log
-func CreateLogEntryHandler(params entries.CreateLogEntryParams) middleware.Responder {
-	ctx := params.HTTPRequest.Context()
-	httpReq := params.HTTPRequest
+func createLogEntry(ctx context.Context, params entries.CreateLogEntryParams) (models.LogEntry, int, string, error) {
```

Comment:
It is a little weird that we return and error and a error message? But I dunno, maybe that was the only sane way to refactor this and it doesn't matter since it's internal?

Reply:
I might be able to return a logEntry, middleware.Responser, and err. I wasn't entirely sure how to chain together middleware.Responders, it was a little opaque to me

Reply:
That seems pretty OK